.gitignore

```diff
@@ -4,6 +4,7 @@ config/*.o
 target/
 Website/.next
 Website/node_modules
+*.db
```

Comment:
@michalryn Drifty already creates a `.drifty` folder (for linux and mac) and `Drifty` folder in Applications folder (probably) in case of windows. I would recommend putting the db file there. Also, it is better to separately name the db as `drifty_cli.db` and `drifty_gui.db`, to prevent data loss.
You can use the `Program` class which gives you the path to the drifty folder; the code would look like this: `Program.get(Program.DRIFTY_PATH)`.